Resources/config/controller.xml

```diff
@@ -17,6 +17,7 @@
             <argument type="service" id="liip_monitor.helper.runner_manager" />
             <argument type="service" id="liip_monitor.helper" />
             <argument>%liip_monitor.view_template%</argument>
+            <argument type="service" id="logger" on-invalid="ignore" />
```

Comment:
Core Symfony appears to only use a NullLogger in testing. Other logging happens within a `if (null !== $this->logger)` check, '[for performance reasons](https://github.com/symfony/symfony/pull/14673#discussion_r30510354)'.

I think that any tiny reduction of speed within the bundle would be more than offset by the clarity of code with a `new NullLogger()` in the constructor - but I'd be happy to wrap the logging calls I've written in conditionals anyway if that is the general consensus.

That would also leave the controller.xml as it was originally - allowing the logger to be able to be optional, and so fail-safe if there was no such service (in Yaml, it would be "?@logger"), and so it would default to `null` from the constructor parameter.